data/core/emptyview.lua

```diff
@@ -15,47 +15,51 @@ function EmptyView:get_filename()
-  local x1, y1 = x, y + ((dh - th) / #lines)
-  local xv = x1
-  local title = "Lite XL"
-  local version = "version " .. VERSION
-  local title_width = style.big_font:get_width(title)
-  local version_width = style.font:get_width(version)
-  if version_width > title_width then
-    version = VERSION
-    version_width = style.font:get_width(version)
-    xv = x1 - (version_width - title_width)
-  end
-  x = renderer.draw_text(style.big_font, title, x1, y1, color)
-  renderer.draw_text(style.font, version, xv, y1 + th, color)
-  x = x + style.padding.x
-  renderer.draw_rect(x, y, math.ceil(1 * SCALE), dh, color)
-  th = style.font:get_height()
-  y = y + (dh - (th + style.padding.y) * #lines) / 2
-  local w = 0
-  for _, line in ipairs(lines) do
-    local text = string.format(line.fmt, keymap.get_binding(line.cmd))
-    w = math.max(w, renderer.draw_text(style.font, text, x + style.padding.x, y, color))
-    y = y + th + style.padding.y
-  end
-  return w, dh
-end
+EmptyView.commands = {
+  { fmt = "%s to run a command", cmd = "core:find-command" },
+  { fmt = "%s to open a file from the project", cmd = "core:find-file" },
+  { fmt = "%s to change project folder", cmd = "core:change-project-folder" },
+  { fmt = "%s to open a project folder", cmd = "core:open-project-folder" },
+}
 
 function EmptyView:draw()
   self:draw_background(style.background)
-  local w, h = draw_text(0, 0, { 0, 0, 0, 0 })
-  local x = self.position.x + math.max(style.padding.x, (self.size.x - w) / 2)
-  local y = self.position.y + (self.size.y - h) / 2
-  draw_text(x, y, style.dim)
+  -- x,y center-point
+  local x = self.position.x + self.size.x / 2
+  local y = self.position.y + self.size.y / 2
+  local divider_w = math.ceil(1 * SCALE)
+  local cmds_x = x + math.ceil(divider_w /2) + style.padding.x
+  local logo_right_side = x - math.ceil(divider_w/2) - style.padding.x
+
+  local displayed_cmds = {}
+  for _, command in ipairs(self.commands) do
+    local keybinding = keymap.get_binding(command.cmd)
+     if keybinding ~= nil then
+      table.insert(displayed_cmds,{
+        fmt = command.fmt,
+        keybinding = keybinding
+      })
+     end
+  end
+  local cmd_h = style.font:get_height() + style.padding.y
+  local cmds_y = y - ((cmd_h * #displayed_cmds)/2)
+  for i, cmd in ipairs(displayed_cmds) do
+    local cmd_text = string.format(cmd.fmt, cmd.keybinding)
+    renderer.draw_text(style.font, cmd_text, cmds_x, cmds_y + cmd_h*(i-1), style.dim)
+  end
+
+  local title = "Lite XL"
+  local version = VERSION
+  local logo_h = style.big_font:get_height(title)
+  local logo_y = y - logo_h + logo_h/4
+  local logo_x = logo_right_side - style.big_font:get_width(title)
+  local vers_x = logo_right_side - style.font:get_width(version)
+  renderer.draw_text(style.big_font, title, logo_x, logo_y, style.dim)
+  renderer.draw_text(style.font, version, vers_x, y, style.dim)
+
+  local divider_y =  math.min(cmds_y, logo_y) - style.padding.y
```

Comment:
The divider line grows with the number of commands but is never smaller than the logo